src/PortingAssistant.Client.Analysis/Utils/PackageCompatibility.cs

```diff
@@ -122,6 +122,22 @@ public static async Task<CompatibilityResult> IsCompatibleAsync(Task<PackageDeta
                     CompatibleVersions = new List<string>()
                 };
             }
+
+            catch (AggregateException ae)
+            {
+                foreach (var e in ae.Flatten().InnerExceptions)
+                {
+                    if (!(e is PortingAssistantClientException))
+                    {
+                        _logger.LogError("parse package version {0} {1} with error {2}", packageVersionPair.PackageId, packageVersionPair.Version, e);
+                    }
```

Comment:
Do we need to handle the case if `e` is not a PortingAssistantClientException?

Reply:
The If statement is for e != PortingAssistantClientException in line 130

Reply:
Thanks, Xuening. I believe I understand what's going on now. 

I think we can remove the `if` statement here and the logging message in my comment below (`PortingAssistantNuGetHandler.cs`, line 105).